test/e2e/tests/notifications/enable-notifications-with-accounts-sync.spec.ts

```diff
@@ -14,12 +14,76 @@ import {
 import NotificationsListPage from '../../page-objects/pages/notifications-list-page';
 import NotificationsSettingsPage from '../../page-objects/pages/settings/notifications-settings-page';
 import SettingsPage from '../../page-objects/pages/settings/settings-page';
+import { Driver } from '../../webdriver/driver';
 import { mockNotificationServices } from './mocks';
 
 describe('Enable Notifications - With Accounts Syncing On', function () {
   this.timeout(120000); // Multiple Syncing features can cause this test to take some time
 
   describe('from inside MetaMask', function () {
+    async function completeOnboardingWithSyncedAccounts(driver: Driver) {
```

Comment:
just a small nit: it seems like a better place to have this functions might be in a file inside `flows`  folder. This way we'll be able to use them in any test we want. What do you think?
Non-blocking, this could be done in a separate PR if we want to

Reply:
Wow, I didn't realise this `flows` folder!

If is is okay, since we will add back a commented out test case, I'll do this in a subsequent PR 😄

Reply:
thank you @Prithpal-Sooriya ! I've approved the PR then. Could you share the subsequent PR once you have it? :pray: